### PR TITLE
Enabling nu != nu_div case

### PR DESCRIPTION
**Pull request by oksanaguba**

### CODE REVIEW

**Comment by oksanaguba** [comment]
On bowman on 1 thread

[onguba@node01 bxx-branch]$ ctest -L dev -R nudiv
Test project /ascldap/users/onguba/runhomme-bowman/bxx-branch
    Start 16: prtcB-r3-nudiv-dry-f-ne2-ndays1
1/3 Test #16: prtcB-r3-nudiv-dry-f-ne2-ndays1 ............***Failed   13.67 sec
    Start 17: prtcB-r3-nudiv-dry-c-ne2-ndays1
2/3 Test #17: prtcB-r3-nudiv-dry-c-ne2-ndays1 ............***Failed    9.68 sec
    Start 81: prtcB-r3-nudiv-dry-ne2-ndays1_cxx_vs_f90
3/3 Test #81: prtcB-r3-nudiv-dry-ne2-ndays1_cxx_vs_f90 ...   Passed    1.67 sec

**Comment by oksanaguba** [comment]
Am i supposed to run clang format on each pr?

File: src/share/cxx/HyperviscosityFunctorImpl.hpp [hunk 1]
```diff
@@ -187,8 +189,9 @@ class HyperviscosityFunctorImpl
 
     if (m_data.nu_top > 0) {
 
+      //for top 3 levels and laplace, there is trivial nu_ratio only
       m_sphere_ops.vlaplace_sphere_wk_contra<NUM_BIHARMONIC_LEV>(
-            kv, m_data.nu_ratio,
+            kv, 1.0,
```
Comment (mfdeakin-sandia) [inline src/share/cxx/HyperviscosityFunctorImpl.hpp]: Not knowing the Fortran, should this be `1.0` instead of one of the `nu_ratio` variables?

**Comment by mfdeakin-sandia** [comment]
Now that we're an open repo, I think we can use tools like travis-ci to automagically run clang-format on master each night? That way we don't have to worry about version differences, and we should be using travis anyways for the unit tests

File: src/share/cxx/HyperviscosityFunctorImpl.hpp (hunk 1 above)
Comment (oksanaguba) [reply src/share/cxx/HyperviscosityFunctorImpl.hpp]: i am not sure what's the question: for top 3 levels we won't use nu_ratios, but they are not always 1... i followed fortran.

Comment (mfdeakin-sandia) [reply src/share/cxx/HyperviscosityFunctorImpl.hpp]: Thanks for the clarification; feel free to merge

Comment (oksanaguba) [reply src/share/cxx/HyperviscosityFunctorImpl.hpp]: Michael is correct that NaN would work better, but i still want to keep 1.0 here...

**Pull request merged**